### DISCUSSION TITLE
Python 3.7 and 3.8

### DISCUSSION THREAD

**Discussion by raysalemi** [tox-dev/tox#2271] [Q&A] [answered]
Hi,

I work on one machine that has Python 3.7 installed and a different one with Python 3.8 installed.  I currently have to modify tox.ini on each machine to set the version of python to match the machine:

```
[tox]
# when changing this list, adapt CONTRIBUTING.md accordingly:
#envlist = py{35,36,37,38,39}-{linux,macos,windows},docs
envlist = py{37, 38}-{linux,macos,windows}
```

This gives an error on my Mac:

```
SKIPPED:  py37-linux: platform mismatch ('darwin' does not match 'linux|cygwin')
ERROR:  py37-macos: InterpreterNotFound: python3.7
SKIPPED:  py37-windows: platform mismatch ('darwin' does not match 'win32')
SKIPPED:  py38-linux: platform mismatch ('darwin' does not match 'linux|cygwin')
  py38-macos: commands succeeded
SKIPPED:  py38-windows: platform mismatch ('darwin' does not match 'win32')
```

To run on one machine I have to remove the `37`. On the other machine, I have to remove the `38`.

Is there a way to skip a combination if the version of Python is not available?

**Comment by gaborbernat** [comment, marked as answer]
https://tox.wiki/en/latest/config.html#conf-skip_missing_interpreters

**Reply by raysalemi** [reply]
Thanks!
